okhttp/src/main/java/com/squareup/okhttp/Cache.java

```diff
@@ -66,9 +66,9 @@
  * valid. Such responses increment both the network count and hit count.
  *
  * <p>The best way to improve the cache hit rate is by configuring the web
- * server to return cacheable responses. Although this client honors all <a
+ * server to return cacheable responses. This client honors all <a
  * href="http://www.ietf.org/rfc/rfc2616.txt">HTTP/1.1 (RFC 2068)</a> cache
- * headers, it doesn't cache partial responses.
```

Comment:
Yeah, we still don't actually do the hard work of caching partial responses properly.